deployability/modules/testing/README.MD

```diff
@@ -0,0 +1,341 @@
+## Testing Module
+
+### User documentation
+
+The execution of the tests is carried out through the Workflow library, or by executing them manually through commands.
+Execution can be done from any operating system.
+
+Initially, Python libraries must be installed. It is recommended to use virtual environments. Follow the technical documentation at https://docs.python.org/3/library/venv.html.
+
+1. Activate the environment:
+
+  ```bash
+  source {venv directory}/bin/activate
```

Comment:
Add the Python command to create the virtual environment

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/5005/commits/66983858545c260b04b271ca0cd647627fc80d08